docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:xenial
+ 
+RUN apt-get update && apt-get install -y curl lsb-release
+RUN curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | bash
+ 
+RUN /opt/google-fluentd/embedded/bin/gem uninstall fluent-plugin-google-cloud
+RUN /opt/google-fluentd/embedded/bin/gem install fluent-plugin-google-cloud -v 0.6.4.pre.3
```

Comment:
Does the `curl` `gem uninstall` and `gem install` need to go in separate layers? Can they be chained?